csrc/quantization/gguf/mmvq.cuh

```diff
@@ -0,0 +1,2172 @@
+#define VDR_Q4_0_Q8_1_MMVQ 2
```

Comment:
Ditto, I assume https://github.com/ggerganov/llama.cpp/blob/dd07a123b79f9bd9e8a4ba0447427b3083e9347a/ggml/src/ggml-cuda/mmvq.cuh and https://github.com/ggerganov/llama.cpp/blob/dd07a123b79f9bd9e8a4ba0447427b3083e9347a/ggml/src/ggml-cuda/mmvq.cu